rethinkdb/datadog_checks/rethinkdb/exceptions.py

```diff
@@ -0,0 +1,15 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+class RethinkDBError(Exception):
```

Comment:
Should extend CheckException? Is there a need for custom exceptions?

Reply:
> Is there a need for custom exceptions?

For simplicity, not really. -> Dropped the `exceptions` module in fc047d554